### PR TITLE
Seeding-script done

### PR DESCRIPTION
**Pull request by jasonschreib**
Some functions I wrote are not used...may use them later though if I ever get a chance to refactor

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,5 @@ coverage
 build/
 dist/
 
+# env file
+.env
```
Comment (maxmilpro) [inline .gitignore:28]: it's good practice to end js files with a new line. for reference: [Newline necessary at the end of JavaScript files?](https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/#:~:text=So%2C%20to%20conclude%3A%20When%20in,you%20should%20put%20a%20newline.)

File: db/aws.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    //TODO = return the response
+  } catch (e) {
+    console.log('Error', e);
+  }
+};
+
+
+module.exports = {
+  getAllObjects
+}
```
Comment (maxmilpro) [inline db/aws.js:75]: add new line

File: db/aws.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  //increment the counter after adding a new normal photo
+  counter++;
+  return `https://${process.env.BUCKET_NAME}.s3.us-west-1.amazonaws.com/normalphoto-${counter}`;
+};
+
+//function to handle upload to S3 of thumbnail photo - must return the new url
+const uploadThumbnailToS3 = async (images) => {
+  console.log('HEY THERE', images[1]);
+  //put the object into S3
+  // await s3.putObject({
```
Comment (maxmilpro) [inline db/aws.js:39]: clean up commented out code

File: db/helpers.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      photos.push(thumbPhoto);
+      console.log(photos);
+      return photos;
+    })
+    .catch(function (error) {
+      console.log('Error', error);
+    });
+};
+
+//helper function to resize photo to thumbnail
```
Comment (maxmilpro) [inline db/helpers.js:29]: recommend removing the extra code. might be a good opportunity to make a trello ticket

File: db/helpers.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+function generateThumbnail() {
+
+};
+
+
+
+module.exports = {
+  generatePhotos,
+  generateThumbnail
+}
```
Comment (maxmilpro) [inline db/helpers.js:40]: add newline

**Review by maxmilpro — COMMENTED** [review]
overall this is great. understand why you had 2 significant progress updates! looks like you learned a lot about aws. it would be good to clean up some of the comments and extraneous code. great work!

follow up questions:
- where are you using uploadNormalToS3 and uploadThumbnailToS3? if i follow your code, i see you use getAllObjects while seeding which retieves all your photos from S3. how are the photos getting into S3 in the first place?

**Comment by jasonschreib** [comment]
Thanks for the code review! I had those 2 functions for uploading to S3, but instead I bypassed that and populated my S3 bucket myself with downloads and then just used the getObjects function. I have it in the backlog to come back to this and retry using those functions

**Pull request merged**